Tools/ARM-Templates/DataConnectors/AzureAD/README.md

```diff
@@ -0,0 +1,17 @@
+# Azure Active Directory connector template
+
+Author: Javier Soriano
+
+This template enables the Azure AD connector on your Sentinel workspace. 
+
+Azure Active Directory Audit/SignIn logs requires permissions to deploy at tenant scope. Assign Owner or Contributor to the principal that needs to deploy the templates (details [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-to-tenant?tabs=azure-cli#required-access)):
```

Comment:
remove the "en-us" from all URLs

Reply:
Thanks for pointing @sarah-yo! It seems to be fine now :)